roles/nextflow/defaults/main.yml

```diff
@@ -3,11 +3,9 @@ nextflow_java: "{{ java_home }}"
 nextflow_version_tag: 21.04.3
 nextflow_download_url: https://github.com/nextflow-io/nextflow/releases/download/v{{ nextflow_version_tag }}/nextflow
 nextflow_env:
-  NXF_LAUNCHER: $HOME/.nextflow/work
-  NXF_LAUNCHBASE: $HOME/.nextflow/work
   NXF_HOME: "{{ nextflow_dest }}/workfiles"
-  NXF_TEMP: $HOME/.nextflow/work
-  NXF_WORK: $HOME/.nextflow/work
+  NXF_TEMP: "{{ ngi_pipeline_nobackup }}/nextflow/tmp"
```

Comment:
Running sarek outside of ngi_pipeline currently works and uses $SNIC_TMP for storing temporary files. I believe it's this part of the config that makes it work: https://github.com/nf-core/configs/blob/d0d86bfb2af26904d81de60d8ab6c8358262d8ea/conf/uppmax.config#L62 is the issue rather that ngi_pipeline doesn't find the nf-core configs?

Reply:
Yeah I get a bit lost here between work, temp and scratch.. I'm not entirely sure what `NXF_TEMP` is used for.. 🤔